src/Chirp.Infrastructure/Repositories/LikeRepository.cs

```diff
@@ -25,13 +27,14 @@ public void AddLike(Like like)
         // Check if like already exists
         if (LikeExists(like))
         {
-            throw new Exception("Cheep is already liked by author!");
+            // This is more resilient idempotency wise since we could get duplicate requests across the network
+            return;
```

Comment:
Please explain this to me like I'm 5

Reply:
When I send mail via the postal service they can duplicate my letter by accident sending it with two different deliverers. When that happens they might both get it to me, but it does not make sense to call the post office and yell at them for receiving it twice, we just throw the letter out.